### PR TITLE
[IDWA-OCR-98] Auto Move Iterations Forward

### PR DESCRIPTION
**Pull request by derekadombek**
# Pull Request

I feel the best way to test this action without creating a new project is just to merge it into main and see if it works at 0500 on Thursday and it it doesn't we'll adjust and fix from there.

#98 
## Description
[Provide a brief description of the changes implemented in this pull request.]

## Related Issues
[Link any related issues or tasks from your project management system.]

## Additional Notes
[Add any additional context or notes that reviewers should know about.]

## Checklist
Please review and complete the following checklist before submitting your pull request:

- [x] I have ensured that the pull request is of a manageable size, allowing it to be reviewed within a single session.
- [x] I have reviewed my changes to ensure they are clear, concise, and well-documented.
- [x] I have updated the documentation, if applicable.
- [ ] I have added or updated test cases to cover my changes, if applicable.
- [x] I have minimized the number of reviewers to include only those essential for the review.
- [x] I have notified teammates in the review thread to build awareness.

## Checklist for Reviewers
Please review and complete the following checklist during the review process:

- [ ] The code follows best practices and conventions.
- [ ] The changes implement the desired functionality or fix the reported issue.
- [ ] The tests cover the new changes and pass successfully.
- [ ] Any potential edge cases or error scenarios have been considered.

### CODE REVIEW

File: .github/workflows/projects-auto-move-new-sprint.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    name: Move to next iteration
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: blombard/move-to-next-iteration@master
+      with:
+        owner: CDCgov
+        number: 33
+        token: ${{ secrets.PROJECTS_MOVE_TO_NEXT_ITERATION }}
+        iteration-field: Iteration
```
Comment (derekadombek) [inline .github/workflows/projects-auto-move-new-sprint.yml:18]: I'm hoping this option is for naming the new sprint and all we have to do is rename it once this action happens but if this is suppose to be the name of the previous sprint then it might cause this action to fail. This public action lacks maturity and documentation which makes it hard to know.

**Review by zdeveloper — APPROVED** [review]
LGTM! thank you!
